File: js files/4a.js
// 4a. Write a program in JS that stores a list of fruits and prints each fruit using a loop.

function displayFruits() {
    const fruits = ["Apple", "Banana", "Orange", "Mango", "Strawberry"];
    
    console.log("List of fruits:");
    for (let fruit of fruits) {
        console.log(`- ${fruit}`);
    }
}

displayFruits();

File: py files/4a.py
# 4a: Write a program in Python that stores a list of fruits and prints each fruit using a loop.

def display_fruits():
    fruits = ["Apple", "Banana", "Orange", "Mango", "Strawberry"]
    
    print("List of fruits:")
    for fruit in fruits:
        print(f"- {fruit}")

if __name__ == "__main__":
    display_fruits()